sets/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Set",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the set of resources."
```

Comment:
also change "set of resources"

Reply:
I changed that from "resource set" to "set of resources" because having it say "Name of the set" seemed a little vague/unclear to me (since a set is a common data structure). Not particularly attached if you feel strongly, though.

Reply:
I think set of recipes makes more sense?

Reply:
Ah, fair - I'll change that.